weave

```diff
@@ -1634,6 +1634,7 @@ case "$COMMAND" in
         protect_against_docker_hang
         VOLUME_CONTAINERS=$(docker ps -qa --filter label=weavevolumes)
         [ -n "$VOLUME_CONTAINERS" ] && docker rm -v $VOLUME_CONTAINERS  >/dev/null 2>&1 || true
+        rm -f $HOST_ROOT/var/lib/weave/weave-netdata.db >/dev/null 2>&1 || true
```

Comment:
I couldn't see any point in having an extra directory there.  

If we expected multiple files it would make sense, although I would name the directory `/var/lib/weave/net` or similar to make the grouping plain.